src/ontology/mondo-edit.obo

```diff
@@ -198989,6 +198990,7 @@ xref: Orphanet:71493 {source="MONDO:subClassOf", source="OMIM:300331"}
 xref: UMLS:C1845888 {source="OMIM:300331", source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
 is_a: MONDO:0019111 {source="MONDOLEX:0010301", source="OMIM:300331", source="ORDO:71493/btnt"} ! familial thrombocytosis
 property_value: confidence "1.1130104463437789" xsd:double
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3172 xsd:string
```

Comment:
could you please add a comment too with the reason for obsoletion and term it should be merged with?
Thanks!

Reply:
done